python/tests/test_dwelling.py

```diff
@@ -148,6 +183,19 @@ def test_from_row(self, sample_input_d: reader.InputData) -> None:
                     area_metres=9.2903,
                     length_metres=3.048,
                 )
+            ],
+            walls=[
+                extracted_datatypes.Wall(
+                    label='Second level',
+                    structure_type_english='Wood frame',
+                    structure_type_french='Ossature de bois',
+                    component_type_size_english='38x89 mm (2x4 in)',
+                    component_type_size_french='38x89 (2x4)',
+                    nominal_rsi=1.432,
+                    effective_rsi=1.8016,
+                    perimeter=42.9768,
+                    height=2.4384,
+                )
```

Comment:
Also needs a test for when `constructionTypeCode` is None

Reply:
Might be better in `test_extracted_datatypes`

Reply:
Such a test has been added to `test_extracted_datatypes`